lib/perl/Genome/Model/Tools/Pindel/RunPindel.pm

```diff
@@ -447,6 +458,25 @@ sub _run_pindel_for_chromosome {
     return $result;
 }
 
+sub _run_pindel_for_region {
+    my ($self, $region_file) = @_;
+    
+    my $output_basename = join '/', $self->output_directory, 'all_sequences';
+    my $window_size     = $self->window_size;
+    my $ref_seq         = $self->reference_sequence_input;
+
+    my $cmd = $self->pindel_path . " -f ".$ref_seq. " -i " . $self->_config_file . " -o ". $output_basename . " -j ".$region_file . " -w ".$window_size." -b /dev/null";
+    Genome::Sys->shellcmd(cmd => $cmd);
```

Comment:
I think it's preferable for new `shellcmd` calls to use the array form of `cmd` instead of the string form when possible, i.e.
```perl
my $cmd = [$self->pindel_path, "-f ", $ref_seq, "-i", $self->_config_file, "-o", $output_basename, "-j", $region_file, "-w", $window_size, "-b", "/dev/null"];
```